azure-pipelines.yml

```diff
@@ -38,6 +38,9 @@ stages:
           - linux: build_docs
             pytest: false
 
+          - linux: mypy
```

Comment:
`mypy` has pre-commit hooks if you want to avoid adding another job here: https://github.com/pre-commit/mirrors-mypy

Reply:
You could have both. It usually helps.

Reply:
I only didn't put it in the pre-commit because mypy can be a little slow to run (and there wasn't any mention of the pre-commit in mypy's docs).

Reply:
Also I in general prefer using pre-commit for stuff which autofixes, especially with the pre-commit ci service autofixing now.